### DISCUSSION TITLE
Postgres extract support?

### DISCUSSION THREAD

**Discussion by PaulAtST** [sequelize/sequelize#16973] [Help & Questions]
Does sequelize support Postgres date extraction function extract? 

For example I can call Postgres DATE_TRUNC function the following way: 
```
attributes: [
    [Sequelize.fn('DATE_TRUNC', 'hour', Sequelize.col('start_time'), 'America/New_York'), 'intervalStart'],
}
```

But when I call extract it fails with an unexpected syntax error: 
```
attributes: [
    [Sequelize.fn('EXTRACT', 'hour', Sequelize.col('start_time'), 'America/New_York'), 'intervalStart'],
}
```

**Comment by ephys** [comment]
`EXTRACT` does not expect a string as the first parameter. You can maybe wrap the `hour` part in `literal`, not sure if that works. In any case, this can be written using the new `sql` tag in Sequelize 7: https://sequelize.org/docs/v7/querying/raw-queries/
